### PR TITLE
Week 2 new

### PR DESCRIPTION
**Pull request by lisettedeputter**

### CODE REVIEW

**Review by Roy-M-Moore — COMMENTED** [review]
The documentation is sufficient and helpful. The tests are also useful and are well structured with the testthat package. Good job.

File: R/GenerateItems.R [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#' Generate Symbol Search Items
```
Comment (rebekka-cebulla) [inline R/GenerateItems.R:1]: I think this is a really clear and useful documentation of your function

File: tests/testthat/test-GenerateItems.R [hunk 2]
```diff
@@ -0,0 +1,57 @@
+library('ProcessingSpeed')
+
```
Comment (rebekka-cebulla) [inline tests/testthat/test-GenerateItems.R:2]: These tests are nice!

**Review by rebekka-cebulla — COMMENTED** [review]
I really like the overall structur of your github pachage page!

**Pull request merged**